cartoframes/data/observatory/catalog/geography.py

```diff
@@ -9,6 +9,7 @@
 from . import utils
 
 GEOGRAPHY_TYPE = 'geography'
+PLATFORM_BQ = 'bq'
```

Comment:
Could we set it globaly. RIght now, it's defined for the two modules

Reply:
I have removed it since we are going to keep only the `CatalogEntity` method